client/my-sites/domains/domain-management/edit/registered-domain.jsx

```diff
@@ -137,7 +137,7 @@ class RegisteredDomain extends React.Component {
 
 		return (
 			<VerticalNavItem path={ path }>
-				{ this.props.translate( 'Name Servers and DNS' ) }
+				{ this.props.translate( 'Name servers and DNS' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.